### PR TITLE
Update Portuguese datetime and Numbers with unit regex

### PR DESCRIPTION
**Pull request by samhickey25**
Update Portuguese datetime regex for SuffixAfterRegex, so it does not match on a period at the end of a datetime input. eg "11:55."

Update Portuguese NumbersWithUnit CompoundUnitConnectorRegex, so that compound sentences for currency will resolve eg "15 dólares e 15 centavos, por favor"

### CODE REVIEW

**Review by tellarin — CHANGES_REQUESTED** [review]
Looks OK to me, but can you add a couple new spec cases to show the change works as intended?
All PRs should have tests that show the change working and that can be used to avoid future regressions.
If possible, can you test if Spanish suffers the same issue and also fix there if needed?
Thanks.

**Comment by samhickey25** [comment]
> 

I've added some specs, and made some changes to they python parsers for number_with_unit. This issue does not exist fro Spanish

File: Specs/NumberWithUnit/Portuguese/CurrencyModel.json [hunk 1]
```diff
@@ -1536,13 +1536,13 @@
...
     "Input": "custou apenas 15 dólares e 15 centavos.",
-    "NotSupported": "javascript, python, java",
+    "NotSupported": "javascript, java",
     "Results": [
       {
         "Text": "15 dólares e 15 centavos",
         "TypeName": "currency",
         "Resolution": {
-          "value": "15,15",
+          "value": "15.15",
```
Comment (tellarin) [inline Specs/NumberWithUnit/Portuguese/CurrencyModel.json:1545]: You should not modify the separators. Please revert these changes so we can merge.

Comment (tellarin) [reply Specs/NumberWithUnit/Portuguese/CurrencyModel.json:1545]: This is a legacy design decision, that hopefully will be finally addressed and uniformly sorted out in release 2.0. But that will take a while.

Comment (samhickey25) [reply Specs/NumberWithUnit/Portuguese/CurrencyModel.json]: No problem, reverted those changes

Comment (tellarin) [reply Specs/NumberWithUnit/Portuguese/CurrencyModel.json]: There is only one revert pending for "3,99" in the output of "3 euros com 99 centavos".

Comment (samhickey25) [reply Specs/NumberWithUnit/Portuguese/CurrencyModel.json]: Ah yes sorry! fixed now

**Review by tellarin — APPROVED** [review]
